heartbeater/heartbeater.go

```diff
@@ -0,0 +1,64 @@
+package heartbeater
```

Comment:
Let's put this under the application package. Since it's used to detect applications.